modules/nginxvts/charts.go

```diff
@@ -0,0 +1,118 @@
+package nginxvts
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+	// Dims is an alias for module.Dims
+	Dims = module.Dims
+)
+
+var nginxVtsMainCharts = Charts{
+	{
+		ID:    "times",
+		Title: "Nginx running time( Uptime=(nowMsec-loadMsec)/1000 )",
+		Units: "milliseconds",
+		Fam:   "main",
+		Ctx:   "nginxvts.times",
+		Dims: Dims{
+			{ID: "loadmsec", Name: "load"},
+			{ID: "nowmsec", Name: "up"},
+		},
+	},
+	{
+		ID:    "connections",
+		Title: "Nginx Connections",
+		Units: "connections",
+		Fam:   "main",
+		Ctx:   "nginxvts.connections",
+		Dims: Dims{
+			{ID: "connections_active", Name: "active"},
+			{ID: "connections_reading", Name: "reading"},
+			{ID: "connections_writing", Name: "writing"},
+			{ID: "connections_waiting", Name: "waiting"},
+			{ID: "connections_accepted", Name: "accepted"},
+			{ID: "connections_handled", Name: "handled"},
+			{ID: "connections_total", Name: "total"},
+		},
+	},
+}
+var nginxVtsSharedZonesChart = Charts{
+	{
+		ID:    "size",
+		Title: "Shared memory size",
+		Units: "bytes",
+		Fam:   "sharedzones",
+		Ctx:   "nginxvts.sharedzones_size",
+		Dims: Dims{
+			{ID: "sharedzones_maxsize", Name: "max size"},
+			{ID: "sharedzones_usedsize", Name: "used size"},
+		},
+	},
+	{
+		ID:    "node",
+		Title: "Number of node using shared memory",
+		Units: "count",
```

Comment:
```suggestion
		Units: "nodes",
```

`count` is to generic, we try to use something more specific then `number`/`count`